util/keyfile/src/bin/sample_keys_main.rs

```diff
@@ -1,13 +1,25 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
-use std::path::PathBuf;
+use hex::FromHex;
+use std::{path::PathBuf, vec::Vec};
 use structopt::StructOpt;
 
+// Hack to work around Vec special handling in structopt
+type VecBytes = Vec<u8>;
```

Comment:
Have you tested that this works? I recall `structopt` wanted you to create a newtype a la `struct VecBytes(Vec<u8>)`, but this is nicer.

Reply:
Yeah - this works :)